microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -32,6 +32,44 @@ class LtpTestsuite(TestSuite):
     LTP_LITE_TESTS = ["math", "ipc", "mm", "sched", "pty", "fs"]
     LTP_REQUIRED_DISK_SIZE_IN_GB = 2
 
+    def _get_parameter(
+        self,
+        variables: Dict[str, Any],
+    ) -> None:
+        # read parameters from .yml file
+        tests = variables.get("ltp_test", "")
+        skip_tests = variables.get("ltp_skip_test", "")
+        self.ltp_source_file = variables.get("ltp_source_file", "")
```

Comment:
Don't use class members in the test suite, it breaks the design. Use a top-level common method, and no output parameters needed.